components/uniset/src/uniset.rs

```diff
@@ -168,14 +179,26 @@ impl UnicodeSet {
     /// assert_eq!(None, example_iter_ranges.next());
     /// ```
     pub fn iter_ranges(&self) -> impl ExactSizeIterator<Item = RangeInclusive<u32>> + '_ {
-        self.inv_list.chunks(2).map(|pair| RangeInclusive::new(pair[0], pair[1] - 1))
+        self.inv_list
+            .chunks(2)
+            .map(|pair| RangeInclusive::new(pair[0], pair[1] - 1))
     }
 
     /// Returns the number of ranges contained in this [`UnicodeSet`]
     pub fn get_range_count(&self) -> usize {
         self.inv_list.len() / 2
     }
 
+    /// Returns a specific range contained in this [`UnicodeSet`] by index.
+    /// Intended for use in FFI.
+    pub fn get_range(&self, idx: usize) -> Option<RangeInclusive<u32>> {
```

Comment:
Suggestion:

```suggestion
    pub fn get_nth_range(&self, idx: usize) -> Option<RangeInclusive<u32>> {
```

Or perhaps:

- `get_range_at`
- `get_range_at_index`
- `range_at_index`